### PR TITLE
Fix/increase unit tests

### PR DESCRIPTION
**Pull request by julia-zack**
We went from 77% coverage to 82%.

The increment was focused on important files, as it say:
- went from 68% cov to 96% in pegout status service,
- from 47% cov to almost 74% in bridge service,
- from 60% to 87.5% on pegout data processor.

### CODE REVIEW

File: src/__tests__/unit/services/pegout-data.processor.unit.ts [hunk 1]
```diff
@@ -810,4 +810,97 @@ describe('Service: PegoutDataProcessor', () => {
...
+    const events: ExtendedBridgeEvent[] = extendedBridgeTx.events as ExtendedBridgeEvent[];
+    const hasEvent = events.find(event => event.name === BRIDGE_EVENTS.RELEASE_REQUESTED);
+    const releaseRequestedEvent = events.find(event => event.name === BRIDGE_EVENTS.RELEASE_REQUESTED);
+
+    expect(hasEvent).true;
+    expect(releaseRequestedEvent).not.null;
+    await thisService['processIndividualPegout'](extendedBridgeTx);
+
+    // im reaching to line 340 with this test.
+    // im facing trouble to test what it follows, i guess related to the TODO comment on line 334.
```
[the remark this answers is not in the document]
Comment (julia-zack) [reply src/__tests__/unit/services/pegout-data.processor.unit.ts:904]: I've tried that but is not working

[the remark this answers is not in the document]
Comment (julia-zack) [reply src/__tests__/unit/services/pegout-data.processor.unit.ts]: done, thanks!!

**Review by lserra-iov — APPROVED** [review]

**Review by ronaldsg20 — APPROVED** [review]
LGTM

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**